### PR TITLE
Upgrade BHoM_Engine to .Net Standard 2.0

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
### NOTE: Depends on 
https://github.com/BHoM/BHoM/pull/1292

### CODE REVIEW

File: Structure_Engine/Create/Loads/GeometricalLineLoad.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public static partial class Create
         [Description("Creates a GeometricalLineLoad with a constant force across the length of the Line.")]
         [InputFromProperty("location")]
         [InputFromProperty("loadcase")]
-        [Input("force", "The force to be applied to the full length of the Line.", typeof(Force))]
+        [Input("force", "The force to be applied to the full length of the Line.", typeof(BH.oM.Quantities.Attributes.Force))]
```
Comment (FraserGreenroyd) [inline Structure_Engine/Create/Loads/GeometricalLineLoad.cs:45]: @IsakNaslundBh FYI

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Review by IsakNaslundBh — COMMENTED** [review]
LGTM!

Compiles with structural adapters, and all tested with them. All looks fine

**Comment by FraserGreenroyd** [comment]
@BHoMBot check serialisation
@BHoMBot check null-handling
@BHoMBot check copyright-compliance
@BHoMBot check dataset-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check dataset-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check null-handling

**Review by michaelhoehn — APPROVED** [review]
LGTM

**Comment by FraserGreenroyd** [comment]
@BHoMBot check project-compliance
@BHoMBot check null-handling
@BHoMBot check serialisation
@BHoMBot check ready-to-merge

**Pull request merged**